README.md

```diff
@@ -117,5 +117,5 @@ MIT
 
 [npm-image]: https://img.shields.io/npm/v/koa-requestid.svg?style=flat-square
```

Comment:
Let's remove `?style=flat-square` from this line so that the badge matches the new one for Travis.